tests/test_notificatie_zrc_to_drc.py

```diff
@@ -0,0 +1,104 @@
+"""
+Test whole notification process:
+1. producer side (zrc):
+- registering kanaal
+- sending notification
+
+2. subscriver side (drc):
+- subscribe to kanaal
+- receive notification
+"""
+import pytest
+from zds_client import ClientAuth
+
+from .constants import CATALOGUS_UUID, ZAAKTYPE_UUID
+
+
+@pytest.mark.incremental
+class TestNotificatie:
+
+    def test_zrc_create_kanaal(self, state, nc_client):
+        # can't run django command register_kanaal
+        # therefore test Client.request under this command
+        nc_client.auth.set_claims(
+            scopes=['notificaties.scopes.publiceren']
+        )
+        # check if kanaal exists - for local testing:
+        kanaal_naam = 'zaken'
+        kanalen = nc_client.list('kanaal', query_params={'naam': kanaal_naam})
+
+        if kanalen:
+            kanaal = kanalen[0]
+        else:
+            kanaal = nc_client.create('kanaal', {'naam': kanaal_naam})
+
+        assert kanaal['naam'] == kanaal_naam
+        assert 'url' in kanaal
+        state.kanaal = kanaal
+
+    def test_drc_subscriber_to_zaken_kanaal(self, state, nc_client, drc_client):
+        # can't add Subscription in the drc admin
+        # therefore test Client.request under this command
+        nc_client.auth.set_claims(
+            scopes=['notificaties.scopes.consumeren']
+        )
+
+        drc_auth = ClientAuth(
+            client_id='nc',
+            secret='nc-to-drc',
+            scopes=['notificaties.scopes.publiceren']
+        )
+
+        # can't reat url from spec, because it isn't added to schema
+        # not working because /callbacks endpoint is not included in drc
+        callback_url = '{}callbacks'.format(drc_client.base_url)
```

Comment:
Feel free to use f-strings for simple interpolation:

```suggestion
        callback_url = f'{drc_client.base_url}callbacks'
```